src/NzbDrone.Core/MediaFiles/MovieImport/Manual/ManualImportService.cs

```diff
@@ -121,7 +121,7 @@ public ManualImportItem ReprocessItem(string path, string downloadId, int movieI
                 SceneSource = SceneSource(movie, rootFolder),
                 ExistingFile = movie.Path.IsParentPath(path),
                 Size = _diskProvider.GetFileSize(path),
-                Languages = (languages?.SingleOrDefault() ?? Language.Unknown) == Language.Unknown ? languageParse : languages,
```

Comment:
We only want to enforce this logic if there is one language and it's unknown, thus the single. We need to fix this another way.